test/base/CustomExportsBaseSpec.scala

```diff
@@ -151,54 +120,4 @@ trait CustomExportsBaseSpec
       .withCSRFToken
   }
 
-  protected def postRequestFormUrlEncoded(uri: String, declarationId: String = "declarationId")(
-    body: (String, String)*
-  ): Request[AnyContentAsFormUrlEncoded] = {
-    val session: Map[String, String] = Map(
-      ExportsSessionKeys.declarationId -> declarationId,
-      SessionKeys.userId -> FakeAuthAction.defaultUser.identityData.internalId.get
-    )
-
-    FakeRequest("POST", uri)
-      .withHeaders(Map(cfg.headerName -> token).toSeq: _*)
-      .withSession(session.toSeq: _*)
-      .withFormUrlEncodedBody(body: _*)
-      .withCSRFToken
-  }
-
-  // FIXME should be promoted to MockExportsCacheService trait
```

Comment:
Did you address this comment? AFAIR those two versions are different

Reply:
Those methods are mocked in the `MockExportCacheService` in exactly the same way